### PR TITLE
[Delivers #166701226] Add SpidaDBResult

### PR DESCRIPTION
**Pull request by spidaKevinGentile**

### CODE REVIEW

File: resources/schema/spidamin/spidadb/referenced_result.schema [hunk 1]
```diff
@@ -4,16 +4,15 @@
...
+        "dateModified"
     ],
     "properties": {
         "id": {
             "type": "string",
             "description": "The unique id of this project component"
         },
         "calcResult": {
-            "$ref": "../../spidacalc/result/result.schema"
+            "$ref": "../../spidacalc/results/results.schema"
```
Comment (jasongarrett) [inline resources/schema/spidamin/spidadb/referenced_result.schema]: is it possible that this could be either a result or results?

Comment (spidaKevinGentile) [reply resources/schema/spidamin/spidadb/referenced_result.schema]: I don't believe so. We're always putting detailed results at `design.analysisDetails.results[]` even for a single result. The URI in this context was never valid.

**Comment by jeremywen** [comment]
looks good

**Comment by jeremywen** [comment]
looks good

**Pull request merged**